.github/workflows/stainless-builds.yml

```diff
@@ -22,6 +22,10 @@ on:
         description: 'PR number to run Stainless build for'
         required: true
         type: number
+      python_url:
```

Comment:
the name `python_url` feels odd. should this be `sdk_install_url`?

Reply:
good point. I can change this